pyrepl/keymaps.py

```diff
@@ -98,37 +99,34 @@
 python_emacs_keymap = comp_emacs_keymap + (
     (r'\n', 'maybe-accept'),
     (r'\M-\n', 'self-insert'))
-    
+
 reader_vi_insert_keymap = tuple(
-    [(c, 'self-insert')
-     for c in map(chr, range(32, 127)) if c <> '\\'] + \
-    [(c, 'self-insert')
-     for c in map(chr, range(128, 256)) if c.isalpha()] + \
-    [(r'\C-d', 'delete'),
-     (r'\<backspace>', 'backspace'),
-     ('')])
+    [(c, "self-insert") for c in map(chr, range(32, 127)) if c != "\\"]
+    + [(c, "self-insert") for c in map(chr, range(128, 256)) if c.isalpha()]
+    + [(r"\C-d", "delete"), (r"\<backspace>", "backspace"), ("")]
+)
 
 reader_vi_command_keymap = tuple(
     [
-    ('E', 'enter-emacs-mode'),
-    ('R', 'enter-replace-mode'),
-    ('dw', 'delete-word'),
-    ('dd', 'delete-line'),
-    
-    ('h', 'left'),
-    ('i', 'enter-insert-mode'),
-    ('j', 'down'),
-    ('k', 'up'),
-    ('l', 'right'),
-    ('r', 'replace-char'),
-    ('w', 'forward-word'),
-    ('x', 'delete'),
-    ('.', 'repeat-edit'), # argh!
-    (r'\<insert>', 'enter-insert-mode'),
-     ] + 
-    [(c, 'digit-arg') for c in '01234567689'] +
-    [])
-   
+        ("E", "enter-emacs-mode"),
+        ("R", "enter-replace-mode"),
+        ("dw", "delete-word"),
+        ("dd", "delete-line"),
+        ("h", "left"),
+        ("i", "enter-insert-mode"),
+        ("j", "down"),
+        ("k", "up"),
+        ("l", "right"),
+        ("r", "replace-char"),
+        ("w", "forward-word"),
+        ("x", "delete"),
+        (".", "repeat-edit"),  # argh!
+        (r"\<insert>", "enter-insert-mode"),
+    ]
+    + [(c, "digit-arg") for c in "01234567689"]
+    + []
```

Comment:
this line does nothing

Reply:
can you explain more? change on around this line came from darker formatter

Reply:
Ya, I saw it was in the original code.  Appending an empty list to a list does nothing.  Since you are modifying these lines, you may as well also remove this line.

Reply:
https://github.com/pypy/pyrepl/pull/48/commits/8669e51fa91eaa5f42a0900c5c08c7f1b9f2f2fe